lib/content-services/src/lib/content-metadata/interfaces/card-view-group.interface.ts

```diff
@@ -20,4 +20,6 @@ import { CardViewItem } from '@alfresco/adf-core';
 export interface CardViewGroup {
     title: string;
     properties: CardViewItem[];
+    editable: boolean;
```

Comment:
should be nullable to prevent breaking changes:

```
editable?: boollean;
expanded?: boolean;
```

Reply:
I have updated it.